rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/mute/muterunner.kt

```diff
@@ -67,7 +67,15 @@ data class ExecutionResult(
 
 fun String.color(success: Boolean) = if (success) this.green() else this.red()
 
-fun File.linkTo(line: Int) = "${this.name}(${this.name}:$line)"
+fun File.linkTo(line: Int) = if (showSourceAbsolutePath()) {
+    // For linking to source from Visual Studio Code console
+    "${this.absolutePath}:$line"
+} else {
+    // For linking to source from IDEA console
+    "${this.name}(${this.name}:$line)"
+}
+
+fun showSourceAbsolutePath(): Boolean = "true" == System.getProperty("showSourceAbsolutePath")
```

Comment:
You could try this:
```suggestion
fun showSourceAbsolutePath() = Boolean.parseBoolean(System.getProperty("showSourceAbsolutePath"))
```

Also, what if that property is not set? Is the default false?

Reply:
I think that that way we could get a NPE if the property is not defined

Reply:
In my idea,  if that property is not set, it should be set to false